src/ocm/components/hosts/HardwareStatus.tsx

```diff
@@ -15,9 +19,10 @@ type HardwareStatusProps = {
   host: Host;
   validationsInfo: ValidationsInfo;
   onEditHostname?: () => void;
+  zIndex?: number;
```

Comment:
I'd prefer to have a prop such as `isWithinModal` and then convert it to whatever the zIndex needs to be in the displaying component.